lib/woothee/crawler.rb

```diff
@@ -88,7 +88,7 @@ def self.challenge_crawlers(ua, result)
       end
     end
     if ua.index('Yeti')
-      if ua.index('http://help.naver.com/robots')
+      if ['http://help.naver.com/robots', 'http://help.naver.com/support/robots.html', 'http://naver.me/bot'].any? { |url| ua.index(url) }
```

Comment:
This code creates Array object every time.
Use just 3 `ua.index(uri)` with `||`, or a constant of 3 URIs.